sksurv/meta/stacking.py

```diff
@@ -285,3 +286,61 @@ def predict_log_proba(self, X):
         """
         Xt = self._predict_estimators(X)
         return self.final_estimator_.predict_log_proba(Xt)
+
+    @property_available_if(_meta_estimator_has("unique_times_"))
+    def unique_times_(self):
```

Comment:
Could you please add a API doc string for `unique_times_`too?

Reply:
Hmm, adding docs somehow doesn't work i.e. it seems the doc generation isn't "aware" of the conditionality of the attribute being documented. So I've for now undone the changes. Interestingly for the conditional methods there doesn't seem to be the same issue it seems.